Tests/CommonMarkTests/NodeHashableTests.swift

```diff
@@ -0,0 +1,64 @@
+import XCTest
+import CommonMark
+
+final class NodeHashableTests: XCTestCase {
```

Comment:
I bumped on the name `NodeHashableTests`, despite `Equatable` indeed being a requirement of `Hashable`. I think I'd prefer `NodeTests`, but I'd probably also want to consolidate the various `Document___Tests` into a single file as well. We can do that in a follow-up; no need to change that here. Any strong preference one way or another?

Reply:
I had similar thoughts, but I wanted to keep the name of the file consistent with the rest of the tests. +1 for merging into a single file.

Also, I added the tests for the `Hashable` implemtation only for `Node`. We should add them for `Document` as well, if we want to have full test coverage for the publicly exposed API of the library. The fact that `Document` inherits from `Node` and does not override the `Hashable` implementation is an implementation detail which could change in the future. Do you have any strong opinion on this?

Reply:
I renamed the file (and class name) to `NodeTests` and organized the tests for the different parts with `// MARK: -` Then we can add more tests for `Node` in the future and unify the tests for `Document` with a later change.

Reply:
> Also, I added the tests for the `Hashable` implemtation only for `Node`. We should add them for `Document` as well, if we want to have full test coverage for the publicly exposed API of the library. The fact that `Document` inherits from `Node` and does not override the `Hashable` implementation is an implementation detail which could change in the future. Do you have any strong opinion on this?

Since `Document` was originally _not_ a subclass of `Node`, I think it's reasonable to have test coverage in case that ever changes again. I'll do that in a follow-up after this gets merged.

> I renamed the file (and class name) to NodeTests and organized the tests for the different parts with // MARK: - Then we can add more tests for Node in the future and unify the tests for Document with a later change.

Perfect. Thanks!